ts/components/startup.ts

```diff
@@ -67,7 +68,7 @@ export type MATHDOCUMENT = MathDocument<any, any, any>;
 export type HANDLER = Handler<any, any, any>;
 export type DOMADAPTOR = DOMAdaptor<any, any, any>;
 export type INPUTJAX = InputJax<any, any, any>;
-export type OUTPUTJAX = OutputJax<any, any, any>;
+export type OUTPUTJAX = OutputJax<any, any, any> & {font: any};
```

Comment:
Is that a legal type?

Reply:
Yes.  It is a join of the two types, and is needed in order to process the `loadAllFontFiles` option at

https://github.com/mathjax/MathJax-src/blob/aed4385da7ee81ea9d62c3304c75567f56437e21/ts/components/startup.ts#L301

But I can cast to `COMMONJAX` on that line instead, if you prefer.